src/components/radio-button-group/RadioButtonGroup.tsx

```diff
@@ -81,11 +81,18 @@ export const RadioButtonGroup = forwardRef<HTMLDivElement, PropsWithLabels<Props
             )}
           </Label>
 
-          {helpText && <HelperText as="span">{helpText}</HelperText>}
+          {helpText && (
+            <HelperText as="span" id={`${id}-helptext`}>
+              {helpText}
+            </HelperText>
+          )}
         </RadioGroupLabelWrapper>
       )}
 
-      <RadioButtonsWrapper horizontal={alignment === RadioButtonGroupAlignment.Horizontal}>
+      <RadioButtonsWrapper
+        {...(helpText ? { 'aria-describedby': `${id}-helptext` } : {})}
+        {...(label ? { 'aria-labelledby': `${id}-label` } : {})}
```

Comment:
Why going with the complex object spreading? `aria-describedby={helpText? 'id' : undefined}` should be fine, an undefined prop won't appear in the DOM...